runtime-modules/content/src/tests/transfers.rs

```diff
@@ -232,290 +453,327 @@ fn accept_transfer_status_fails_with_non_channel_owner() {
+            )
         );
     })
 }
 
 #[test]
-fn accept_transfer_status_succeeds_for_curators_to_members_with_price() {
+fn accept_channel_transfer_fails_with_invalid_transfer_id() {
     with_default_mock_builder(|| {
-        run_to_block(1);
-
-        create_initial_storage_buckets_helper();
-        increase_account_balance_helper(DEFAULT_MEMBER_ACCOUNT_ID, INITIAL_BALANCE);
-        increase_account_balance_helper(SECOND_MEMBER_ACCOUNT_ID, INITIAL_BALANCE);
-        create_default_curator_owned_channel(DEFAULT_DATA_OBJECT_STATE_BLOAT_BOND, &[]);
-
-        let price = 100;
-        UpdateChannelTransferStatusFixture::default()
-            .with_origin(RawOrigin::Signed(LEAD_ACCOUNT_ID))
+        ContentTest::with_member_channel().setup();
+        InitializeChannelTransferFixture::default()
             .with_new_member_channel_owner(SECOND_MEMBER_ID)
-            .with_actor(ContentActor::Lead)
-            .with_price(price)
             .call_and_assert(Ok(()));
 
-        let member2_balance = Balances::<Test>::usable_balance(&SECOND_MEMBER_ACCOUNT_ID);
-        <Test as Config>::ContentWorkingGroup::set_budget(INITIAL_BALANCE);
-
         AcceptChannelTransferFixture::default()
             .with_origin(RawOrigin::Signed(SECOND_MEMBER_ACCOUNT_ID))
-            .with_price(price)
-            .call_and_assert(Ok(()));
-
-        assert_eq!(
-            <Test as Config>::ContentWorkingGroup::get_budget(),
-            INITIAL_BALANCE + price
-        );
-        assert_eq!(
-            Balances::<Test>::usable_balance(&SECOND_MEMBER_ACCOUNT_ID),
-            member2_balance - price
-        );
+            .with_transfer_id(TransferId::from(2u32))
+            .call_and_assert(Err(
+                Error::<Test>::InvalidChannelTransferCommitmentParams.into()
+            ))
     })
 }
 
 #[test]
-fn accept_transfer_status_succeeds_for_members_to_curators_with_price() {
+fn cancel_channel_transfer_fails_with_invalid_channel_id() {
     with_default_mock_builder(|| {
-        run_to_block(1);
-
-        create_initial_storage_buckets_helper();
-        increase_account_balance_helper(DEFAULT_MEMBER_ACCOUNT_ID, INITIAL_BALANCE);
-        create_default_member_owned_channel();
+        ContentTest::with_member_channel().setup();
+        InitializeChannelTransferFixture::default()
+            .with_new_member_channel_owner(SECOND_MEMBER_ID)
+            .call_and_assert(Ok(()));
 
-        let curator_group_id = Content::next_curator_group_id();
+        CancelChannelTransferFixture::default()
+            .with_channel_id(ChannelId::from(2u32))
+            .call_and_assert(Err(Error::<Test>::ChannelDoesNotExist.into()))
+    })
+}
 
-        let price = 100;
-        UpdateChannelTransferStatusFixture::default()
-            .with_new_channel_owner(ChannelOwner::CuratorGroup(curator_group_id))
-            .with_price(price)
-            .call_and_assert(Ok(()));
+#[test]
+fn cancel_channel_transfer_fails_with_non_member_owner() {
```

Comment:
As mentioned in the previous comment, cases like:
- `cancel_channel_transfer_fails_with_non_member_owner`
- `cancel_channel_transfer_fails_with_lead_and_member_channel`
- `cancel_channel_transfer_fails_with_invalid_curator`

Should be covered by:
- `unsuccessful_curator_channel_actions_with_invalid_context` and
- `unsuccessful_member_channel_actions_with_invalid_context`

(you just need to update `run_all_fixtures_with_contexts` accordingly to one of the previous comments)